web_ir_actions_act_window_none/README.rst

```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+===========
+Noop action
+===========
+
+This module was written to have an action that does nothing. A notorious case
+where this is useful is buttons on popup windows, because they need to return
+an action. Given the default here is `ir.actions.act_window_close`, you'll
+have to jump through some hoops if you have a button that is not supposed to
+close the popup window.
+
+Usage
+=====
+
+To use this module, depend on it in your module and return::
+
+    {'type': 'ir.actions.act_window.none'}
+
+The button will trigger a reload of the popup's form, in case you want to
+suppress this, set key ``reload`` to ``False`` in the action you return.
```

Comment:
Assuming you set a `none` action with `reload=False`...... What's this for? An addon that lets buttons do nothing?

You should drop support of `reload=False` and call this `web_action_reload`, I'm not in favor of merging an addon that adds a feature that does nothing, honestly 😕.